internal/test/resources.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	certv1 "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
 	certMeta "github.com/cert-manager/cert-manager/pkg/apis/meta/v1"
+	"github.com/cryostatio/cryostat-operator/api/v1beta1"
```

Comment:
This package is already imported under `operatorv1beta1`.

Reply:
Removed